swssconfig/swssconfig.cpp

```diff
@@ -39,9 +43,28 @@ void dump_db_item(KeyOpFieldsValuesTuple &db_item)
     SWSS_LOG_DEBUG("]");
 }
 
+ProducerStateTable &get_producer_table(const std::string &table_name)
+{
+    auto iter = table_map.find(table_name);
+    if (iter != table_map.end())
+    {
+        return iter->second;
+    }
+
+    auto ret = table_map.emplace(table_name, ProducerStateTable(&pipeline, table_name, true));
```

Comment:
>ProducerStateTable [](http://example.com/codeflow?start=45&length=18)

You may check this example: [https://github.com/Azure/sonic-swss-common/blob/92f54111172138a3c1b2ed2dfbdda23e84d9ae9d/common/dbinterface.cpp#L326](https://github.com/Azure/sonic-swss-common/blob/92f54111172138a3c1b2ed2dfbdda23e84d9ae9d/common/dbinterface.cpp#L326)

Reply:
Hi, not sure I understand the comment. What do you want me to check?

Reply:
I see code improved s you must already check it :)